rs/ic_os/config/src/types.rs

```diff
@@ -58,12 +58,12 @@ pub struct GuestOSSettings {
     /// When given, this provides the initial state of the registry.
     /// If not given, the node will fetch (initial) registry state from the NNS.
     pub ic_registry_local_store_path: Option<PathBuf>,
-    pub guestos_dev: GuestosDevConfig,
+    pub guestos_dev_settings: GuestosDevSettings,
 }
 
 /// GuestOS development configuration. These settings are strictly used for development images.
 #[derive(Serialize, Deserialize, Debug, PartialEq, Default, Clone)]
-pub struct GuestosDevConfig {
+pub struct GuestosDevSettings {
```

Comment:
nit: Could we apply more consistency in the naming? Currently, we use Guestos but ICOS

Reply:
Good note! Updated to GuestOSDevSettings